aead/Cargo.toml

```
@@ -20,6 +20,7 @@ generic-array = { version = "0.14", default-features = false }
 # optional dependencies
 blobby = { version = "0.3", optional = true }
 heapless = { version = "0.7", optional = true, default-features = false }
+bytes = { version = "1.1.0", optional = true }
```

Perhaps...

```suggestion
bytes = { version = "1.1.0", optional = true, default-features = false }
```

...which will avoid linking `std`